test/unit-test/main.tf

```diff
@@ -14,8 +14,41 @@ module "s3_with_AES256" {
   providers = {
     aws.bucket-replication = aws
   }
-  bucket_prefix = "s3-bucket"
+  bucket_prefix = "unit-test-bucket"
   force_destroy = true
   sse_algorithm = "AES256"
   tags          = local.tags
 }
+
+data "aws_iam_policy_document" "topic" {
+  statement {
+    effect = "Allow"
+
+    principals {
+      type        = "Service"
+      identifiers = ["s3.amazonaws.com"]
+    }
+
+    actions   = ["SNS:Publish"]
+    resources = ["arn:aws:sns:*:*:s3-event-notification-topic"]
+
+  }
+}
+resource "aws_sns_topic" "topic" {
+  name   = "s3-event-notification-topic"
+  policy = data.aws_iam_policy_document.topic.json
+}
+
+module "s3_with_notification" {
+  #checkov:skip=CKV_AWS_300: "Ensure S3 lifecycle configuration sets period for aborting failed uploads - This is not needed in our tests"
+  source = "../.."
```

Comment:
Are you referencing the module locally here when it should be a git repo or something like 

`source = "../../modules/s3_thingy"`

Reply:
It's locally. I copied the previous versions of the code for the source